src/test/java/htsjdk/tribble/util/ParsingUtilsTest.java

```diff
@@ -143,6 +164,18 @@ private void tstExists(String path, boolean expectExists) throws IOException{
     }
 
     @Test
```

Comment:
Good idea. I've added a test dependency on https://github.com/google/jimfs, and a few tests that use non-file URIs that exercise the NIO file path.